cz.zeropage.Formiko.json

```diff
@@ -0,0 +1,300 @@
+{
+    "app-id": "org.flatpak.Formiko",
```

Comment:
This should match the filename. Do you own the domain you used there?

Reply:
fixed :-)